app/src/pages/playground/playgroundUtils.ts

```diff
@@ -230,7 +230,26 @@ export function getTemplateMessagesFromAttributes({
       messages: null,
     };
   }
-
+  if (provider === "ANTHROPIC") {
+    const { success, data } =
+      modelConfigWithInvocationParametersSchema.safeParse(parsedAttributes);
+    if (success) {
+      const messages = inputMessages.data.llm.input_messages;
+      const systemPrompt = data.llm.invocation_parameters["system"];
```

Comment:
```suggestion
      const systemPrompt = data.llm.invocation_parameters?.system;
```

It is less common to reference keys via string unless we don't know the string ahead of time, or it contains characters that can't be represented by a symbol

Reply:
thanks! this is fixed